src/test/java/uk/gov/pay/adminusers/resources/IntegrationTest.java

```diff
@@ -13,6 +13,7 @@
 public class IntegrationTest {
 
     static final String USERS_RESOURCE_URL = "/v1/api/users";
+    static final String USER_EMAILS_RESOURCE_URL = USERS_RESOURCE_URL+"/emails";
```

Comment:
`static final String USER_EMAILS_RESOURCE_URL = "/v1/api/users/emails"` would be consistent with the rest of the file